src/util/Constants.ts

```diff
@@ -347,6 +348,61 @@ export const CommandResponses = {
 				],
 				inline: true
 			});
+	},
+	LOOKUP: (address: IPData) => {
+		if (address.status === 'success') {
+			return new MessageEmbed()
+				.setColor(SentinelColors.LIGHT_BLUE)
+				.addFields({
+					name: 'Address',
+					value: address.query,
+					inline: true
+				}, {
+					name: 'ISP',
+					value: address.isp,
+					inline: true
+				}, {
+					name: 'Country',
+					value: address.country,
+					inline: true
+				}, {
+					name: 'Region',
+					value: address.regionName,
+					inline: true
+				}, {
+					name: 'City',
+					value: address.city,
+					inline: true
+				}, {
+					name: 'Timezone',
+					value: address.timezone,
+					inline: true
+				}, {
+					name: 'ZIP',
+					value: address.zip,
+					inline: true
+				}, {
+					name: 'Latitude',
+					value: address.lat,
+					inline: true
+				}, {
+					name: '	Longitude',
+					value: address.lon,
+					inline: true
+				}, {
+					name: 'ORG',
+					value: address.org,
+					inline: true
+				})
+				.setFooter('Sentinel')
+				.setTimestamp();
+		} else if (address.status === 'fail') {
+			return new MessageEmbed()
+				.setColor(SentinelColors.LIGHT_BLUE)
+				.setDescription('ERROR! Invalid address')
+				.setFooter('Sentinel')
+				.setTimestamp();
+		}
```

Comment:
this can be removed, we already check if the request was successful before passing the data here

this also means the above if block can just be removed, and just return the embed directly